test/alternator/conftest.py

```diff
@@ -15,6 +15,8 @@
 import re
 
 from test.alternator.util import create_test_table, is_aws, scylla_log
+from test.conftest import testpy_test_fixture_scope
+from test.cqlpy.conftest import host  # add required fixtures
```

Comment:
I have mixed feelings regarding the dependency between the conftest from different sibling directories. Do we really have no other choice?

Reply:
There are 2 another options: add all Python one-node suites to one sub-folder and add all this stuff to a common `conftest.py`, or go with the initial approach in this PR and add them to `test/conftest.py`. 

First option is out of scope of this PR, as for the second and current one I don't have preferences. Both work.

Reply:
I got you, thanks for clarification.

Reply:
I don't understand what is this "host" fixture (not a config option - a fixture?), or how the heck it's relevant to Alternator. Anyway, to connect to Alternator, a "host" is not enough, you need a full URL.